src/raiden_libs/utils.py

```diff
@@ -97,3 +101,60 @@ def _filter_presence(event: Dict[str, Any], presence_update_id: int) -> None:
                 self._presence_listener(event, presence_update_id)
 
         return _filter_presence
+
+    def _presence_listener(self, event: Dict[str, Any], presence_update_id: int) -> None:
+        """
+        Update cached user presence state from Matrix presence events.
+
+        Due to the possibility of nodes using accounts on multiple homeservers a composite
+        address state is synthesised from the cached individual user presence states.
+
+        This is a copy from the super class with a slight change.
+        Any user will automatically added to the whitelist whenever the PFS receives
+        a presence update. We do this since we can assume that the PFS needs to know
+        about all raiden users.
+        """
+        if self._stop_event.ready():
+            return
+
+        user_id = event["sender"]
+
+        if event["type"] != "m.presence" or user_id == self._user_id:
+            return
+
+        address = address_from_userid(user_id)
+
+        # Not a user we've whitelisted, skip. This needs to be on the top of
```

Comment:
How does `address is None` relate to being whitelisted? All raiden users should return a non-None value from `address_from_userid`, shouldn't they?

Reply:
Maybe the comment is misleading as I had to change the if condition below. The function is mainly a copy from its superclasse but the if condition below. I changed the comment.